src/Components/test/E2ETest/Infrastructure/ServerFixtures/ServerFixture.cs

```diff
@@ -40,6 +40,16 @@ public ServerFixture()
 
         protected static string FindSampleOrTestSitePath(string projectName)
         {
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("helix")))
+            {
+                var dir = Path.Combine(AppContext.BaseDirectory, projectName);
```

Comment:
@captainsafia this line used to be hardcoded to basictestapp which maybe was the issue with not resolving files properly...fingers crossed

Reply:
The CI gods will let us know soon! *fingers crossed*

Reply:
Track record would say prepare to be smitten then 😭